src/structures/StageChannel.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const BaseGuildVoiceChannel = require('./BaseGuildVoiceChannel');
+
+/**
+ * Represents a guild stage channel on Discord.
+ * @extends {GuildChannel}
+ */
+class StageChannel extends BaseGuildVoiceChannel {
+  _patch(data) {
+    super._patch(data);
+
+    /**
+     * The topic of the stage channel
+     * @type {?string}
+     */
+    this.topic = data.topic;
+  }
+
+  /**
+   * Sets the RTC region of the channel.
+   * @param {?string} region The new region of the channel. Set to `null` to remove a specific region for the channel
+   * @returns {Promise<StageChannel>}
+   * @example
+   * // Set the RTC region to europe
+   * stageChannel.setRTCRegion('europe');
+   * @example
+   * // Remove a fixed region for this channel - let Discord decide automatically
+   * stageChannel.setRTCRegion(null);
+   */
+  setRTCRegion(region) {
+    return this.edit({ rtcRegion: region });
+  }
```

Comment:
Why isn't this on BaseGuildVoiceChannel? VoiceChannels can have this property too

Reply:
Dupe of SpaceEEC's review, I didn't see it before posting mine